redact/tools/redact_file.py

```diff
@@ -20,23 +20,14 @@
 def redact_file(file_path: str, out_type: OutputType, service: ServiceType, job_args: Optional[JobArguments] = None,
                 licence_plate_custom_stamp_path: Optional[str] = None, custom_labels_file_path: Optional[str] = None,
                 redact_url: str = settings.redact_url_default, out_path: Optional[str] = None,
-                api_key: Optional[str] = None, skip_existing: bool = True, save_labels: bool = False,
-                auto_delete_job: bool = True):
+                api_key: Optional[str] = None, ignore_warnings: bool = False, skip_existing: bool = True,
+                save_labels: bool = False, auto_delete_job: bool = True):
     """
     If no out_path is given, <input_filename_redacted> will be used.
     """
 
     # input and output path
     file_path = normalize_path(file_path)
-
-    # videos to archives
-    if Path(file_path).suffix[1:].lower() in VID_EXTENSIONS and out_type == OutputType.archives:
-        out_path = Path(file_path.parent).joinpath(f'{file_path.stem}_redacted.{ARCHIVE_EXTENSIONS[0]}')
-
-    # archives to videos
-    elif Path(file_path).suffix[1:].lower() in ARCHIVE_EXTENSIONS and out_type == OutputType.videos:
-        out_path = Path(file_path.parent).joinpath(f'{file_path.stem}_redacted.{VID_EXTENSIONS[0]}')
-
```

Comment:
This is removed because it overwrites the given `out_path` argument (which it shouldn't). The fact that output does not have the right file extension should be fixed in _redact_.